earth_enterprise/src/fusion/portableglobe/shared/bounds_tracker.cpp

```diff
@@ -0,0 +1,117 @@
+// Copyright 2019 Google Inc.
```

Comment:
This would need a different copyright: 2019 Open GEE Contributors.
Rule of thumb: new code --> 2019 Open GEE Contributors. Existent code modified in a PR, add to existent copyright the Open notice: 2019 Open GEE Contributors.